spiceaidocs/docs/api/http/search.md

```diff
@@ -67,3 +67,62 @@ Response
   "duration_ms": 42,
 }
 ```
+
+### Chunking
```

Comment:
I think this should be in a Search section, and referenced from the API section?